File: backend/src/main/java/org/jdk/project/dto/digitaltwin/DigitalTwinPassengerDto.java
package org.jdk.project.dto.digitaltwin;

import java.util.List;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class DigitalTwinPassengerDto {
  private Long id;
  private String userName;
  private String cardNo;
  private String flightNo;
  private String flightId;
  private String estmTakeOffTime;
  private String latestOffStatus;
  private Long regionId;
  private String roomCode;
  private String coordinate;
  private String memLevel;
  private List<DigitalTwinWarningDto> warningLogList;
}

File: backend/src/main/kotlin/org/jdk/project/dto/digitaltwin/DigitalTwinPassengerDto.kt
package org.jdk.project.dto.digitaltwin

data class DigitalTwinPassengerDto(
    var id: Long? = null,
    var userName: String? = null,
    var cardNo: String? = null,
    var flightNo: String? = null,
    var flightId: String? = null,
    var estmTakeOffTime: String? = null,
    var latestOffStatus: String? = null,
    var regionId: Long? = null,
    var roomCode: String? = null,
    var coordinate: String? = null,
    var memLevel: String? = null,
    var warningLogList: List<DigitalTwinWarningDto>? = null,
)
